anitya_schema/anitya_schema/tests/test_schema.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+"""Unit tests for the message schema."""
+import json
+import os
+import unittest
+
+from fedora_messaging import message
+
+from . import FIXTURES_DIR
+
+
+class ExistingMessagesTests(unittest.TestCase):
+    """
+    Assert schema successfully handle legacy fedmsg messages.
+
+    The fixtures are generated by downloading messages from datagrepper.
+    """
+
+    def test_valid(self):
```

Comment:
I don't see any assert in this method.

Reply:
Yep, that's expected, although I should probably add a comment to that effect. ``validate()`` raises an exception if the message is invalid which would cause the test to fail. The assertion is a basically there there isn't an exception.